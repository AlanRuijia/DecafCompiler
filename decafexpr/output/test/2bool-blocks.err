output prefix: /Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2bool-blocks
llvmas: /usr/local/Cellar/llvm38/3.8.0/lib/llvm-3.8/bin/llvm-as
llc: /usr/local/Cellar/llvm38/3.8.0/lib/llvm-3.8/bin/llc
cc: gcc
codegen: /Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/answer/decafexpr
stdlib: /Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/answer/decaf-stdlib.c
generating llvm code:/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/answer/decafexpr ... ok
; ModuleID = 'C'

define i32 @main() {
entry:
  ret i32 0
}
assembling to bitcode:/usr/local/Cellar/llvm38/3.8.0/lib/llvm-3.8/bin/llvm-as "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2bool-blocks.llvm" -o "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2bool-blocks.llvm.bc" ... ok
converting to native code:/usr/local/Cellar/llvm38/3.8.0/lib/llvm-3.8/bin/llc "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2bool-blocks.llvm.bc" -o "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2bool-blocks.llvm.s" ... ok
linking:gcc -o "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2bool-blocks.llvm.exec" "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2bool-blocks.llvm.s" "/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/answer/decaf-stdlib.c" ... ok
running:/Users/Sei/SFU/CMPT379/DecafCompiler/decafexpr/output/test/llvm/2bool-blocks.llvm.exec ... ok
