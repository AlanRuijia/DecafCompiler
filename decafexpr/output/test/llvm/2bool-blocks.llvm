; ModuleID = 'C'

define i32 @main() {
entry:
  %main = alloca i32
  %x = alloca i1
  %y = alloca i32
  %q = alloca i32
  %p = alloca i32
  %y1 = alloca i1
}

define i32 @main.1() {
entry:
  ret i32 0
}
