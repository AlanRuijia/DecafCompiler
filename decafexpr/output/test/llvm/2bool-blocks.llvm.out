Debug message: Generating program...
Debug message: Generating Extern List...
Debug message: Generating extern function...
17
Debug message: Generating package...
Debug message: Generating FieldDecls...
Debug message: Generating MethodDecl...
Generating MethodDecl Heads...main
Defined variable in line 4 : x
Codegen for Assign to x
Defined variable in line 9 : y
Codegen for Assign to y
Defined variable in line 7 : p
Codegen for Assign to p
Defined variable in line 7 : q
Codegen for Assign to q
Debug message: Generating method call... 
print_int
